### ISSUE TITLE
Mouse wheel constantly sends an up input

### ISSUE THREAD

**Issue by chungy** [86Box/86Box#4787]
### What happened?

When enabling a mouse wheel, there is a constant "up" input being sent as long as the mouse is captured, making most software very annoying or impossible to use.

Tested on a Windows 2000 VM, and 86Box versions back to 4.0 to the current release.

### Configuration file

```ini
[General]
vid_renderer = qt_software
host_cpu = AMD Ryzen 9 5950X 16-Core Processor
uuid = fb157e2a-56ae-5734-85b2-03dd96af318b
dpi_scale = 0
video_filter_method = 0
scale = 4

[Machine]
machine = 5ax
cpu_family = pentium_tillamook
cpu_speed = 300000000
cpu_multi = 4.5
cpu_use_dynarec = 1
fpu_softfloat = 0
time_sync = local
fpu_type = internal
mem_size = 262144

[Video]
gfxcard = px_vision968_pci

[Input devices]
mouse_type = ps2

[Sound]
fm_driver = nuked
sndcard = sbawe64

[Network]
net_01_link = 0
net_02_link = 0
net_03_link = 0
net_04_link = 0

[Storage controllers]
hdc = internal
cassette_mode = load

[Hard disks]
hdd_01_parameters = 63, 16, 20805, 0, ide
hdd_01_fn = disk1.img
hdd_01_speed = 2000_7200rpm
hdd_01_ide_channel = 0:0

[Floppy and CD-ROM drives]
fdd_01_type = 35_2ed
fdd_02_type = 525_2hd
cdrom_01_speed = 52
cdrom_01_parameters = 1, atapi
cdrom_01_ide_channel = 1:0
cdrom_01_type = 86BOX_CD-ROM_1.00

[Standard PS/2 Mouse]
buttons = 4
```

### Operating system

Arch Linux

### CPU

AMD Ryzen 9 5950X

### 86Box version

4.2.1

### Build architecture

Linux - ARM (64-bit)

### Build type

- [ ] New recompiler
- [ ] Debug build

### Download source

I got 86Box from a third party repository (please tell us where)

### Additional context

86box installed via AUR.

**Comment by chungy** [comment]
I bisected for a bit and it seems c1613776cea3b2d76277888e352ba9a52dc5ce05 might have caused this.

**Comment by OBattler** [comment]
That just changed whether or not to invert the deltas.

**Comment by chungy** [comment]
Fixed in 16f4ebab9ca5a1a65bb53a77b406a2fafea52415
